src/release/breaking-changes/insert-content-text-input-client.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Insert Content Text Input Client
+description: Add a new method to the TextInputClient interface to allow Android virtual keyboards to insert rich content into Flutter TextFields. 
+---
+
+## Summary
+
+Added an `insertContent` method to the `TextInputClient` interface to allow Android's image keyboard 
+feature to insert content into a Flutter `TextField`.
+
```

Comment:
Could you make a format operation to the document? I saw there are multiple blank lines that don't follow the regular behavior in this repo.

Reply:
Is this some sort of command I need to run in the terminal? Sorry, not super familiar with what tools are available when creating markdown docs.

Reply:
Maybe a format action from editors? Such as VS Code and IDEA, or other markdown editors should support formatting too.